yarn-project/end-to-end/src/e2e_account_contracts.test.ts

```diff
@@ -56,10 +53,10 @@ const itShouldBehaveLikeAnAccountContract = (
         address,
       };
 
-      ({ logger, pxe, teardown, aztecNode } = await setup(0, { initialFundedAccounts: [accountData] }));
-      wallet = new TestWalletInternals(pxe, aztecNode);
+      ({ logger, wallet, teardown, aztecNode } = await setup(0, { initialFundedAccounts: [accountData] }));
+      wallet = new TestWalletInternals(wallet.getPxe(), aztecNode);
 
-      const accountManager = await AccountManager.create(wallet, pxe, secret, accountContract, salt);
+      const accountManager = await AccountManager.create(wallet, wallet.getPxe(), secret, accountContract, salt);
```

Comment:
The `.getPxe()` getter is a temporary workaround that will get resolved once Wallet instantiates the PXE.